lib/explorer/data_frame.ex

```diff
@@ -4443,10 +4444,10 @@ defmodule Explorer.DataFrame do
   """
   @doc type: :single
   @spec describe(df :: DataFrame.t()) :: DataFrame.t()
-  def describe(df) do
+  def describe(df, percentiles \\ @default_percentiles) do
```

Comment:
Let's make this an option, as we will likely have other percentiles in the future!

Reply:
Note we only need to make it an option here. It does not have to be an option in the behaviour.